include/openssl/bio.h

```diff
@@ -382,6 +384,11 @@ struct bio_dgram_sctp_prinfo {
 };
 # endif
 
+typedef struct bio_buf_mem {
```

Comment:
this struct should be in bio_lcl.h, don't make it public.  and only put the typedef struct line in ossl_types.h